### PR TITLE
(DOCSP-16408): Expose connection string for deployments for easy of use

### LINKED ISSUE #452: [questions] Seed lists?
Does this operator expose seed list DNS anywhere? Since that would make it easier to connect to an auto scaling replica set.

### PR DESCRIPTION
**Pull request by jwilliams-mongo**
[JIRA](https://jira.mongodb.org/browse/DOCSP-16408)

Closes #452 

### All Submissions:

* [ x] Have you opened an Issue before filing this PR?
* [x ] Have you signed our [CLA](https://www.mongodb.com/legal/contributor-agreement)?
* [ x] Have you signed all of your commits?
* [ x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [ x ] Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).

### CODE REVIEW

File: docs/deploy-configure.md
Comment (MongoCaleb) [inline docs/deploy-configure.md:30]: Duplicate "where"s

File: docs/deploy-configure.md
Comment (MongoCaleb) [inline docs/deploy-configure.md:42]: This hyperlink is not rendering properly.  Perhaps because it is in a code block?

**Review by MongoCaleb — APPROVED** [review]
2 things that need fixing, but otherwise LGTM

File: docs/deploy-configure.md
Comment (chatton) [inline docs/deploy-configure.md:28]: there can be multiple users so I believe this should be `that contain users' connection strings` (I could be mistaken!)

**Review by chatton — APPROVED** [review]
LGTM! I just had one grammar note - I am not an expert so could be mistaken, if I will leave it up to you to correct if necessary and merge, thanks for the work on this! 👍

**Review by tibulca — APPROVED** [review]

File: docs/deploy-configure.md
Comment (jwilliams-mongo) [reply docs/deploy-configure.md]: @chatton I had it this way at first. Your comment affirms that I should go with that. updated.

**Pull request merged**
